CoverMyDotNet/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+
+// Information about this assembly is defined by the following attributes.
+// Change them to the values specific to your project.
+
+[assembly: AssemblyTitle ("CoverMyMeds")]
+[assembly: AssemblyDescription ("")]
+[assembly: AssemblyConfiguration ("")]
+[assembly: AssemblyCompany ("")]
```

Comment:
Should this be CoverMyMeds?  Do you know if there are any published guidelines / best practices for setting these values in published assemblies??